### DISCUSSION TITLE
Downloading reanimated:downloadBoost

### DISCUSSION THREAD

**Discussion by 1mehdifaraji** [software-mansion/react-native-reanimated#3567] [Q&A]
I see something weird about reanimated package and that is it always downloads some stuff when generating an apk with react-native ... why it isn't included in the `node_modules` so it won't download everytime ?

<img width="717" alt="Screen Shot 1401-06-22 at 06 39 59" src="https://user-images.githubusercontent.com/63982703/189792080-b34d9d4e-68a2-4e95-b7dc-9324e4fb3510.png">

    "react-native": "0.66.1",
    "react-native-reanimated": "^2.10.0",

**Comment by tomekzaw** [comment]
See #3523.
